src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/PagedBufferedTextWriter.cs

```diff
@@ -83,11 +83,21 @@ public override void Write(char value)
 
         public override void Write(char[] buffer)
         {
+            if (buffer == null)
+            {
+                return;
+            }
+
             Write(buffer, 0, buffer.Length);
         }
 
         public override void Write(char[] buffer, int index, int count)
         {
+            if (buffer == null)
+            {
+                return;
+            }
```

Comment:
Ah, you mean just this overload should throw. Seems reasonable since that's what the other writers do. I'll also take the other methods null checking structure. Functionally it'll be the same but it can't hurt to standardize.